docs/UsersGuide/cmake/cmake-api.md

```diff
@@ -26,7 +26,6 @@ define types and headers needed for F´ for any embedded system they desire.
 [Platform Template](../api/cmake/platform/platform-template.md): Platform file template documentation  
 [Linux](../api/cmake/platform/Linux.md): Linux platform support  
 [Darwin](../api/cmake/platform/Darwin.md): Darwin (macOS) platform support  
-[rtems5](../api/cmake/platform/rtems5.md): RTEMS 5 initial support  
```

Comment:
As noted, the link checker didn't like this link. I did spend a bit of time trying to follow the history of its introduction/removal, but gave up.